### PR TITLE
Use AssertJ instead of JUnit assertions.

### PR DESCRIPTION
**Pull request by indyaah**
- Initial switch from JUnit Assertions to AssertJ

### CODE REVIEW

**Comment by michael-burwig** [comment]
Hi @indyaah, thank you for your code and time - those changes look great to me.

My apologies for the inconveniences wrt merge-conflict-hell due to 2 bigger refactorings/features. We are still reviewing #200 and will most likely merge it within the first half of tomorrow (CEST). Unfortunately this will create some more conflicts on this PR.

If this does not deter you, your PR will still be more than welcome.

**Comment by indyaah** [comment]
@michael-burwig : No worries at all, once 200 is in I will fix the conflicts.

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/structure/file/FileTest.java [hunk 1]
```diff
@@ -73,8 +73,9 @@ public void checkWrite() throws IOException {
 
     byte[] writtenBytes = Files.readAllBytes(file.getFileOnDisk().toPath());
 
-    assertArrayEquals(bytes, writtenBytes);
-    assertEquals(1, outputFile.listFiles().length);
+    assertThat(writtenBytes).isEqualTo(bytes);
+    assertThat(outputFile.listFiles()).isNotNull();
+    assertThat(outputFile.listFiles().length).isEqualTo(1);
```
Comment (Bananeweizen) [inline services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/structure/file/FileTest.java:78]:
```suggestion
    assertThat(outputFile.listFiles()).isNotNull().hasSize(1);
```

File: services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/S3PublisherTest.java [hunk 2]
```diff
@@ -60,7 +61,7 @@ public void publishTestFolderOk() throws IOException, GeneralSecurityException,
 
     List<S3Object> s3Objects = objectStoreAccess.getObjectsWithPrefix("version");
 
-    assertEquals(5, s3Objects.size());
+    assertThat(s3Objects.size()).isEqualTo(5);
```
Comment (Bananeweizen) [inline services/distribution/src/test/java/app/coronawarn/server/services/distribution/objectstore/S3PublisherTest.java:64]:
```suggestion
    assertThat(s3Objects).hasSize(5);
```

**Pull request merged**